docs/en/connector-v2/source/Gitlab.md

```diff
@@ -26,6 +26,8 @@ Used to read data from Gitlab.
 | format                      | String | No       | json          |
 | params                      | Map    | No       | -             |
 | body                        | String | No       | -             |
+| json_field                  | Config | No       | -             |
```

Comment:
It seems `json_field` is `Object` type.

Reply:
![1670133207186](https://user-images.githubusercontent.com/45089228/205476911-86646601-a4ab-4355-92cb-8299bf206741.png)

Please give us some advice. @liugddx

Reply:
> ![1670133207186](https://user-images.githubusercontent.com/45089228/205476911-86646601-a4ab-4355-92cb-8299bf206741.png)
> 
> Please give us some advice. @liugddx

It is `config` type.

Reply:
> > ![1670133207186](https://user-images.githubusercontent.com/45089228/205476911-86646601-a4ab-4355-92cb-8299bf206741.png)
> > Please give us some advice. @liugddx
> 
> It is `config` type.

Thanks.